src/applications/edu-benefits/0994/content/bankInformation.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
 import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
```

Comment:
Can we update to the `va-additional-info` web component: https://design.va.gov/storybook/?path=/docs/components-va-additional-info--default

Reply:
It probably should be updated, but maybe as a separate smaller issue instead of piggybacking off of this task. (Not really related)